### PR TITLE
[OTE][release] Fix  a failure when the empty label is presented in annotation in hi-cls

### PR DESCRIPTION
**Pull request by sovrasov**
- Empty label is added to the multilabel head

### CODE REVIEW

**Comment by sovrasov** [comment]
run ote-test

**Comment by goodsong81** [comment]
run ote-test

**Comment by sovrasov** [comment]
run ote-test

**Comment by sovrasov** [comment]
run ote-test

**Comment by sovrasov** [comment]
@yunchu I also made a small fix related to tests. Please, have a look

**Comment by sovrasov** [comment]
run ote-test

**Comment by sovrasov** [comment]
run ote-test

File: external/deep-object-reid/torchreid_tasks/utils.py [hunk 1]
```diff
@@ -256,8 +256,8 @@ def generate_label_schema(not_empty_labels: List[LabelEntity], multilabel: bool
 
 
 @check_input_parameters_type()
-def get_multihead_class_info(label_schema: LabelSchemaEntity):
-    all_groups = label_schema.get_groups(include_empty=False)
+def get_multihead_class_info(label_schema: LabelSchemaEntity, with_empty=False):
```
Comment (JihwanEom) [inline external/deep-object-reid/torchreid_tasks/utils.py:259]: `with_empty` makes me little confused. How about to use `include_empty`? I'm okay with `label_schema.get_labels(include_empty=include_empty)`

**Pull request closed without merging**